snow/consensus/snowball/flat.go

```diff
@@ -11,11 +11,10 @@ import (
 var _ Consensus = (*Flat)(nil)
 
 func NewFlat(params Parameters, choice ids.ID) Consensus {
-	f := &Flat{
-		params: params,
+	return &Flat{
+		params:        params,
+		nnarySnowball: newNnarySnowball(params.BetaVirtuous, params.BetaRogue, choice),
```

Comment:
Super nitty - but I try to initialize in the order that the struct defines
```suggestion
		nnarySnowball: newNnarySnowball(params.BetaVirtuous, params.BetaRogue, choice),
		params:        params,
```